server/events/command_runner.go

```diff
@@ -447,7 +453,21 @@ func (c *DefaultCommandRunner) runProjectCmdsParallel(cmds []models.ProjectComma
 
 func (c *DefaultCommandRunner) runProjectCmds(cmds []models.ProjectCommandContext, cmdName models.CommandName) CommandResult {
```

Comment:
I see this is only implemented for the serial implementation, any reason?

Reply:
Nope, I wasn't aware of there being also a parallel implementation. I don't think we use it at my workplace.